backend/src/main/java/com/empire/Region.java

```diff
@@ -371,6 +358,30 @@ public boolean isSea() {
 		return type == Type.WATER;
 	}
 
+	Set<Integer> getCloseRegionIds(World w, int limit) {
```

Comment:
Could you maybe throw in a quick docstring for this method since it is more on the algorithmic end.  Just a quick what it does and how it works so we don't have to re-understand it periodically.

Reply:
Done.